functions.j2

```diff
@@ -33,10 +33,10 @@ function process_command_exit_status() {
 }
 
 {# Macro to be used by the loops below #}
-{% macro function(function, file_extensions, tools, allow_filter, version_argument, validations, security_tools) %}
+{% macro function(function, file_extensions, security_tools, allow_filter, version_argument, validations) %}
```

Comment:
`tools` and `security_tools` were the exact same input before, accidentally duplicated.  So this is just deduplication, no functional difference.